copier.yml

```diff
@@ -22,17 +22,25 @@ package_short_description:
     type: str
     placeholder: "A short description of your package"
     help: What is your package short description?
-author_name:
+author_first_name:
     type: str
-    placeholder: "Your full name"
-    help: What is your full name?
+    placeholder: "Guilherme"
+    help: What is the author's given name?
+author_family_name:
+    type: str
+    placeholder: "Castelão"
+    help: What is your family name?
 email:
     type: str
-    placeholder: "Your email address"
+    placeholder: "gpimenta@nrel.gov"
     help: What is your email address?
+orcid:
+    type: str
+    placeholder: "https://orcid.org/0000-0000-0000-0000"
+    help: What is your ORCID? (optional)
 repository:
     type: str
-    placeholder: "https://github.com/NREL/{{ project_slug }}"
+    default: "https://github.com/NREL/{{ project_slug }}"
```

Comment:
Ok wait, this is interesting! You use the `project_slug` here, but shouldn't this be `project_name`? I'm thinking about `reVRt` or `COMPASS` as examples. Their slugs are all lowercase but the URL has upper case letters. The same can even be said for this repo!

Reply:
Good point. The `project_name` must be a valid repo name, right?

Again, the `PyMFI` and `mfi` are a simpler case that don't even deal with hyphens. I think you're right. Good catch!